releases/unreleased/support-authenticated-queries.yml

```diff
@@ -0,0 +1,14 @@
+---
+title: Support authenticated queries
+category: added
+author: Miguel Ángel Fernández <mafesan@bitergia.com>
+issue: 35
+notes: >
+    After this change, users can perform authenticated
+    queries to Bestiary. For doing so, they must
+    generate a JWT (JSON Web Token) which has to be
+    included within the `Authorization` header in the
+    HTTP requests. 
```

Comment:
This text shouldn't be like a commit message, so don't start it with `After this change`. I would be better something like `Users will be able to perform authenticated queries to Bestiary` or `This feature allows to perform authenticated queries to Bestiary`.